libraries/holochain-service-client/src/main/aidl/org/holochain/androidserviceruntime/holochain_service_client/IHolochainServiceApp.aidl

```diff
@@ -6,7 +6,7 @@ import org.holochain.androidserviceruntime.holochain_service_client.ZomeCallUnsi
 
 interface IHolochainServiceApp {
     void setupApp(IHolochainServiceCallback callback, in InstallAppPayloadFfiParcel request, boolean enableAfterInstall);
-    void enableApp(IHolochainServiceCallback callback, String installedAppId);
-    void ensureAppWebsocket(IHolochainServiceCallback callback, String installedAppId);
+    void enableApp(IHolochainServiceCallback callback);
+    void ensureAppWebsocket(IHolochainServiceCallback callback);
```

Comment:
Unrelated fix -- no need to pass the installedAppId to individual app binding calls. Each app binding is scoped to a single installedAppId.